lib/main.dart

```diff
@@ -20,37 +27,63 @@ class SeedsApp extends StatefulWidget {
   _SeedsAppState createState() => _SeedsAppState();
 }
 
-class _SeedsAppState extends State<SeedsApp> {
-  final AuthService authService = AuthService();
 
+class _SeedsAppState extends State<SeedsApp> {
   @override
   Widget build(BuildContext context) {
-    return ToolboxApp(
-      noItemsFoundWidget: Padding(
-        padding: const EdgeInsets.all(32),
-        child: SvgPicture.asset(R.noItemFound),
-      ),
-      child: MaterialApp(
-        debugShowCheckedModeBanner: false,
-        theme: ThemeData(
-          primarySwatch: primary,
-          tabBarTheme: TabBarTheme(
-            indicator: TabRoundedLineIndicator(
-              context,
-              indicatorColor: primary,
-            ),
-          ),
+    return MultiProvider(
+      providers: [
+        Provider.value(value: HttpService()),
```

Comment:
```suggestion
        Provider(create: (_) =>  HttpService()),
```
see https://pub.dev/packages/provider#usage